src/query/xasl_cache.c

```diff
@@ -954,6 +954,15 @@ xcache_find_xasl_id_for_execute (THREAD_ENTRY * thread_p, const XASL_ID * xid, X
 
   assert ((*xcache_entry) != NULL);
 
+  /* set xasl_id to tde before getting locks */
+  int tran_index = LOG_FIND_THREAD_TRAN_INDEX (thread_p);
+  LOG_TDES *tdes_p = LOG_FIND_TDES (tran_index);
+  assert (tdes_p != NULL);
+  if (tdes_p != NULL && xid != NULL)
```

Comment:
라인 위쪽에서 xid 사용을 보았을 때 NULL은 불가능해 보입니다.
```suggestion
  if (tdes_p != NULL)
```